charts/prometheus-operator-admission-webhook/Chart.yaml

```
@@ -2,7 +2,7 @@
 apiVersion: v2
 description: Prometheus Operator Admission Webhook
 name: prometheus-operator-admission-webhook
-version: 0.19.1
+version: 0.19.2
```

The change should be considered a new feature:

```suggestion
version: 0.20.0
```